bindsnet/network/nodes.py

```diff
@@ -79,8 +79,8 @@ class Input(Nodes):
     Layer of nodes with user-specified spiking behavior.
     """
 
-    def __init__(self, n: Optional[int] = None, shape: Optional[Iterable[int]] = None, traces: bool = False,
-                 trace_tc: float = 5e-2) -> None:
+    def __init__(self, n: Optional[int] = None, shape: Optional[Iterable[int]] = None,
+                 traces: bool = False, trace_tc: float = 5e-2, real_value_input: bool = False) -> None:
```

Comment:
Can you remove this for now? I'm going to refactor this in a `RealInput` object.